### ISSUE TITLE
chpl fails w/ PGI-17.4

### ISSUE THREAD

**Issue by PHHargrove** [chapel-lang/chapel#7091]
Does the failure below (seen using chapel's master and PGI compilers on x86-64 Linux) look familiar to anybody?  
I can provide more info if needed.  
-Paul

```
$ chpl -o hello6-taskpar-dist hello6-taskpar-dist.chpl
PGC-S-0155-Empty initializer not supported  (/run/user/19214/chpl-phargrov-1370.deleteme/chpl__defn.c: 408)
PGC/x86-64 Linux 17.4-0: compilation completed with severe errors
```

**Comment by daviditen** [comment]
Hi Paul,

That error was caused by PR #7050 (merged on 8/21) and should be fixed with PR #7074 (merged on 8/22).

Thanks for catching it.

**Comment by daviditen** [comment]
PR #7074 fixed this, so closing.
